documentation/tut/building_uis.rst

```diff
@@ -178,71 +178,77 @@ The result looks like this in your browser:
 
 
  
-From FlexMeasures, we are using the `[GET] /assets <../api/v2_0.html#get--api-v2_0-assets>`_ endpoint, which loads a list of assets. Note how, unlike the user endpoint above, we are passing a query parameter here (``owner_id``). We are only displaying a subset of the information which is available about assets. Browse the endpoint documentation to learn other information you could get.
+From FlexMeasures, we are using the `[GET] /assets <../api/v3_0.html#get--api-v3_0-assets>`_ endpoint, which loads a list of assets.
+Note how, unlike the user endpoint above, we are passing a query parameter here (``owner_id``).
+We are only displaying a subset of the information which is available about assets.
+Browse the endpoint documentation to learn other information you could get.
 
+For a listing of public assets, replace `/api/v3_0/assets` with `/api/v3_0/assets/public`.
 
-Embedding plots
-------------------------
 
-Creating plots from data can consume lots of development time. FlexMeasures can help here by delivering ready-made plots.
+Embedding charts
+------------------------
 
-In this tutorial, let's display two plots: one with power measurements and forecasts (a solar panel installation) and one with schedules of several EV chargers on the same location, next to each other for easy comparison.
+Creating charts from data can consume lots of development time.
+FlexMeasures can help here by delivering ready-made charts.
+In this tutorial, we'll embed a chart with electricity prices.
```

Comment:
Actually, from the screenshot it seems that this is not just a chart that displays a fixed date range, but it comes with a date picker. So it's quite interactive.

For another time/PR: Can we leave the datepicker out, or auto-hide it?

Reply:
But it doesn't come with a date picker, though. I just took the image from the toy tutorial, where it is embedded in our own asset page. I tried to make it clear by adding the comment `(but then just the chart)`.

Reply:
Mh, I see how you did an 80/20 approach here. Maybe I'll get to it as well (to show what the developer would really get)

Reply:
I already made the picture, maybe you can add it?

![image](https://user-images.githubusercontent.com/30658763/206484298-ae487bd2-386a-4d64-9e5f-4cec5cba278c.png)